features/nationaliteithistorie.feature

```diff
@@ -12,16 +12,38 @@ Functionaliteit: Tonen van Nationaliteithistorie
   # Elk categorievoorkomen bevat de gegevens die vanaf de ingangsdatum gelden plus de registratiedatum.
   # Voor een beëindigde nationaliteit zit in de actuele categorie geen nationaliteit, omdat die op dat moment niet meer geldig is. De ingangsdatum in de actuele categorie is dan de datum vanaf wanneer de persoon deze nationaliteit niet meer had.
 
+  # Geprobeerd scenario's zodanig te schrijven met als doel het beschrijven/documenteren van de gewenste functionaliteit
 
   Rule: Voor een actueel geldende nationaliteit of bijzonder Nederlanderschap wordt de datumIngangGeldigheid gevuld met de datum geldigheid (85.10) in de actuele categorie (04).
-
+      
     Scenario: actueel en ongewijzigde nationaliteit
       Gegeven de ingeschreven persoon met burgerservicenummer 000009830 kent de volgende nationaliteiten:
         | Stapel | Categorie | nationaliteit (05.10) | reden opnemen (63.10) | reden beëindigen (64.10) | bijzonder Nederlanderschap (65.10) | onjuist (84.10) | datum ingang geldigheid (85.10) |
         | 1      | 4         | 0001                  | 001                   |                          |                                    |                 | 19750707                        |
       Als de nationaliteithistorie met burgerservicenummer 000009830 wordt geraadpleegd
       Dan bevat het nationaliteithistorie-item met nationaliteit.code "0001" het gegeven "datumIngangGeldigheid" met datum "1975-07-07"
 
+    # Vervangt bovenstaande scenario
+    # kolommen zijn weggelaten die niet toedoen of niet relevant zijn voor deze scenario. Bijv. stapel, categorie, reden opnemen en lege velden
+    # wat wordt bedoeld met actueel?
+    # Wat wordt bedoeld met ongewijzigd? Wordt 'geen historische gegevens' bedoeld?
```

Comment:
Of zelfs: had in het verleden geen andere nationaliteit(en) dan de huidige nationaliteit? Of geldt dit ook als bijvoorbeeld de ingangsdatum oid is veranderd?
Navragen bij RvIG?